engine/src/main/scala/cromwell/engine/workflow/WorkflowManagerActor.scala

```diff
@@ -108,10 +109,11 @@ class WorkflowManagerActor(config: Config)
      */
     case Event(SubmitWorkflowCommand(source), stateData) =>
       val updatedEntry = submitWorkflow(source, replyTo = Option(sender), id = None)
-      // Submit success brought in from the Cromwell API handler
-      sender ! WorkflowManagerSubmitSuccess(updatedEntry.workflowId)
       stay() using stateData.withAddition(updatedEntry)
-    case Event(AbortWorkflowCommand(id), stateData) =>
+    case Event(SubscribeToWorkflowCommand(id), data) =>
+      data.workflows.get(id) foreach {_ ! SubscribeTransitionCallBack(sender())}
+      stay()
+    case Event(AbortWorkflowWMACommand(id), stateData) =>
```

Comment:
`AbortWorkflowWMACommand`: oh my!  What a terrible name!  I'll switch it to `WorkflowManagerActor.AbortWorkflowCommand`.  There were also 2 other `AbortWorkflowCommand`s in the code and I just didn't want to change them all at the beginning because I didn't know which ones would survive after I was done.